platforms/ucl_myriad/install

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -l
+
+# Myriad version / Rachel version
+
+#initial install of everything using pip, wget and git clone
+#also currently runs cath-af pytest and nextflow hello example
+
+SCRIPT_DIR=~/repos/cath-alphaflow/platforms/ucl_myriad
+source ${SCRIPT_DIR}/include
+
+#venv setup, install dependencies
+source ~/.bashrc
+module load python3
+python3 -m venv ${CATH_VENV_PATH}
+. ${CATH_VENV_PATH}/bin/activate
+
+source ~/.bashrc
+module load python3
+module load java/openjdk-11/11.0.1
+pip install --upgrade pip wheel pytest nextflow
```

Comment:
here the pip nextflow module installs a command "nextflow" into the venv which, when run, automatically installs nextflow for you and replaces itself with actual nextflow, and also ensures nextflow in the path, therefore if you install this pip module you shouldn't need to also curl install nextflow as well

Reply:
Yes thanks, my oversight, I'll remove the pip install.